docs/sources/k6-studio/set-up/install.md

```diff
@@ -6,24 +6,41 @@ weight: 100
 
 # Install k6 Studio
 
-You can install k6 Studio on Windows and macOS.
+You can install k6 Studio on Windows, macOS, and Linux.
 
 ## Before you begin
 
 - Install [Google Chrome](https://www.google.com/chrome/) on your computer.
```

Comment:
@Llandy3d with Linux support, do we need to add something specific about Chromium here, or to the Linux section?

Reply:
I would add a note that on Linux `arm64` only Chromium is supported (because Chrome doesn't exists) 🤔